release-notes.txt

```diff
@@ -257,6 +257,8 @@ All code files contain licensing information.
 
 
 
+
+
```

Comment:
remove?

Reply:
This happened during merge, but there are five lines above "RIOT-2017.04 - Release Notes" as well, so is this really a problem?

Reply:
Indeed, it's not the top priority problem of this PR.

Reply:
This should be fixed now.